custom-routes-vis/src/main/webapp/script.js

```diff
@@ -28,10 +28,34 @@ const MarkerNames = {
  * @enum {string}
  */
 const url_ids = {
-  IOS_URL_ID: 'ios-url', 
+  IOS_DEV_URL_ID: 'ios-dev-url', 
+  IOS_PRODUCTION_URL_ID: 'ios-production-url',
   V1_ANDROID_URL_ID: 'v1-android-url', 
   V2_ANDROID_URL_ID: 'v2-android-url',
 };
+
+/**
+ * Container ids for HTML elements that allow users to display an origin marker.
+ * @enum {string}
+ */
+const originDisplayMarkerContainerIds = {
+  LAT_ID: 'origin-lat', 
+  LNG_ID: 'origin-lng',
+  SHOW_CUSTOM_MARKER_ID: 'show-origin-custom-marker', 
+  SHOW_MARKER_ID: 'show-origin-marker',
+};
+
+/**
+ * Container ids for HTML elements that allow users to display a destination marker.
+ * @enum {string}
+ */
+const destinationDisplayMarkerContainerIds = {
+  LAT_ID: 'destination-lat', 
+  LNG_ID: 'destination-lng',
+  SHOW_CUSTOM_MARKER_ID: 'show-destination-custom-marker', 
+  SHOW_MARKER_ID: 'show-destination-marker',
+};
+
```

Comment:
This seems extra but I guess theres no better way to do it

Reply:
Yea I agree, that's why I originally messaged you and jiehua about it, but if others have ideas, feel free to chime in (this is for reducing the repetitive .style.display = 'block', code below @tzhong92 @carmenBen @ChenyuHeidiZhang